### ISSUE TITLE
Messages appearing on wrong streams

### ISSUE THREAD

**Issue by joaopsys** [kilianc/rtail#120]
Hi, I don't know if rtail is still under development, but I might as well just report this anyway:

At the moment I have 10 streams and sometimes I randomly get lines from 1 stream into another stream. They vanish as soon as I switch streams back and forth (they go to their correct stream).

For each stream, my process list shows this:

`tail -F log1.log`
`node --harmony /usr/bin/rtail -m --name log1`

I wonder if I'm doing anything wrong. Anyone getting this issue?

**Comment by joaopsys** [comment]
I'm also feeling this has something to do with the timestamp ordering. Since my streams all have different timestamps (almost different timezones)

**Comment by joaopsys** [comment]
Nope, nothing to do with the timestamps, I set --parse-date false in every rtail, and the issue still remains.

**Comment by dnmgns** [comment]
I'm also experiencing this issue (using latest version).
